Makefile

```diff
@@ -10,7 +16,9 @@ build-traffic-generator:
 
 openshift-deploy:
 	@echo About to deploy a few sample projects to OpenShift
-	./hack/openshift-deploy.sh -n kiali-test-depth -c hack/test-configs/test-depth.yaml
-	./hack/openshift-deploy.sh -n kiali-test-breadth -c hack/test-configs/test-breadth.yaml
-	./hack/openshift-deploy.sh -n kiali-test-hourglass -c hack/test-configs/test-hourglass.yaml
-	./hack/openshift-deploy.sh -n kiali-test-circle-callback -c hack/test-configs/test-circle-callback.yaml
+	ansible-playbook ./test-service/deploy/ansible/deploy_test_meshes.yml -e number_of_services=${NUM_SERVICES} -e number_of_versions=${NUM_VERSIONS} -e ${MESHES} -v
+
+
+openshift-deploy-all-meshes:
```

Comment:
can you create some targets for each individual mesh? So a user can pick the ones to use? i.e.

```
openshift-deploy-test-depth:
    ansible-playbook ./test-service/deploy/ansible/deploy_test_meshes.yml -e number_of_services=${NUM_SERVICES} -e number_of_versions=${NUM_VERSIONS} -e '...whatever it should be...'
openshift-deploy-test-circle:
   ...the ansible command here....

...and other targets...
```

Otherwise, the user will be forced to remember the json syntax for the MESH env var - and I already forgot it :) I don't want to have to remember all this:

MESHES= '{"meshes": ["kiali-test-depth"]}' make openshift-deploy

Reply:
Created as you told on 688672